diffuzers/cli/run_app.py

```diff
@@ -106,6 +122,10 @@ def run(self):
             "--",
             "--device",
             self.device,
+            "--ssl-certfile",
+            self.ssl_certfile,
```

Comment:
does it even work with streamlit? :)

Reply:
Crap honestly totally forgot to test with streamlit, need to refer to their docs for a few things today anywho 
Can follow back up with a few fixes such as tls reqs and fixes for atreamlit